scripts/drone_build_nightly_grafana.sh

```diff
@@ -27,8 +25,10 @@ dagger run --silent go run ./cmd \
   -a exe:grafana:windows/amd64 \
   -a docker:grafana:linux/amd64 \
   -a docker:grafana:linux/arm64 \
+  -a docker:grafana:linux/arm/v7 \
```

Comment:
Not sure if we should add these docker artifacts to all the other builds aswell. Right now they are only set in this script

Reply:
I think building the everywhere is the better approach. The verification and release scripts/steps within grafana/grafana will have to be updated, though (but they need to be anyway ^_^)

Reply:
Sounds good, will add them to the other scripts as well!